src/WebJobs.Script.WebHost/Middleware/ClrOptimizationMiddleware.cs

```diff
@@ -86,7 +92,7 @@ private void StartStopGCAsBestEffort()
             }
             catch (Exception ex)
             {
-                _logger.LogError(ex.Message);
+                _logger.LogError(ex.Message, ex.StackTrace);
```

Comment:
We should be able to use the `LogError` that takes the exception here, which would ensure stack and all other exception details are logged by the providers.

Reply:
good point, updated the PR.